utils/jsonSigning.js

```diff
@@ -5,10 +5,24 @@ const rsasign = require('jsrsasign') // eslint-disable-line no-unused-vars
 const ServerError = require('../errors/server-error')
 const _ = require('lodash')
 
+// Constants for crypto algorithm types (ECDSA and RSA)
+const ES256 = 'ES256'
+const PS256 = 'PS256'
+module.exports.types = {
```

Comment:
This is a stylistic point, but you can do
```
module.exports.types = 
  ES256,
  PS256
}
```
Since you declared the variables above.